### DISCUSSION TITLE
Query by double condition

### DISCUSSION THREAD

**Discussion by yamadaOliva** [pocketbase/pocketbase#5274] [Q&A] [answered]
```javascript
await $app
    .dao()
    .db()
    .select("id")
    .from("family_request")
    .where(
      $dbx.and(
        $dbx.exp("requestFrom = {:id}", { id: userId }),
        $dbx.exp("requestTo = {:id}", { id: familyId })
      )
    )
    .andWhere($dbx.exp("sys_setStatus = 'pending'"))
    .andWhere($dbx.exp("request_status = 'open'"))
    .all(result);

``` 
result.length = 0 even though this value exists in the database

**Comment by ganigeorgiev** [comment, marked as answer]
This is not documented very well at the moment but the name of the binded parameters must be uniquem otherwise in the above the `requestFrom` and `requestTo` expressions will be compared to the same id (probably the last one).

You can either assign different parameter names (eg. `{:toId}` and `{:fromId}`) OR simply replace your `$dbx.and` expression with [`$dbx.hashExp`](https://pocketbase.io/docs/js-database/#dbx-hashexppairs):

```js
$dbx.hashExp({
    requestFrom: userId,
    requestTo:   familyId,
})
```

**Reply by yamadaOliva** [reply]
Thank sir for that answer. It really worked.
